src/libraries/System.Net.Http/src/System/Net/Http/GlobalHttpSettings.cs

```diff
@@ -26,12 +26,12 @@ internal static class SocketsHttpHandler
                 "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP2SUPPORT",
                 true);
 
-            // Default to allowing draft HTTP/3, but enable that to be overridden
-            // by an AppContext switch, or by an environment variable being set to false/0.
-            public static bool AllowDraftHttp3 { get; } = RuntimeSettingParser.QueryRuntimeSettingSwitch(
-                "System.Net.SocketsHttpHandler.Http3DraftSupport",
-                "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP3DRAFTSUPPORT",
-                true);
+            // Default to disable HTTP/3 and QUIC, but enable that to be overridden
+            // by an AppContext switch, or by an environment variable being set to true/1.
+            public static bool AllowHttp3AndQuic { get; } = RuntimeSettingParser.QueryRuntimeSettingSwitch(
+                "System.Net.SocketsHttpHandler.Http3AndQuicSupport",
+                "DOTNET_SYSTEM_NET_HTTP_SOCKETSHTTPHANDLER_HTTP3ANDQUICSUPPORT",
```

Comment:
Was the name of the switch discussed/approved somewhere? I just wonder why it was changed

Reply:
> I just wonder why it was changed

To express that it's used for S.N.Quic as well, which the original switch wasn't. Since S.N.Quic is "hidden" and shouldn't be used, it's not strictly needed. Not even I myself am 100% sure about the naming here :smile: 

About discussing it, it's happening here. AFAIK, switches are not strictly considered public API (e.g. #47583) so no need to go through API Review.

Reply:
Renamed as discussed offline.